src/openms_gui/source/VISUAL/TVToolDiscovery.cpp

```diff
@@ -41,6 +41,7 @@
 #include <QCoreApplication>
 
 #include <iostream>
+#include <thread>
```

Comment:
This addition was accidental (But might be necessary and should have its own PR)

Reply:
k thanks. I can add this manually to develop

Reply:
So when I compiled OpenMS directly from develop without any changes, I got an error. I am using gcc11. I havn't looked into it.

Reply:
Yeah these include hierarchies inside STL seem to change once in a while.
I added that line on develop.